### DISCUSSION TITLE
Search for blog?

### DISCUSSION THREAD

**Discussion by ghost** [facebook/docusaurus#3713] [Q&A] [answered]
it seems algolia only search for doc not including blog part。

**Comment by slorber** [comment, marked as answer]
Hi,  this has been fixed recently with the contextual search feature and work for the RN website, so try to upgrade to latest v2 alpha.

https://reactnative.dev/

![image](https://user-images.githubusercontent.com/749374/98562213-15c5c780-22aa-11eb-95c5-68f59c9d0ca1.png)
